package.json

```diff
@@ -40,7 +39,7 @@
     "jsdom": "3.0.0",
     "jshint": "^2.6.3",
     "react": "^0.14.0",
-    "react-router": "^1.0.3",
+    "react-router": "^2.4.0",
```

Comment:
@JustinMGaiam Thanks for letting us know. What were the errors you were getting?

@samsel I just realized that for those still using `react-router@1`, I may need to update the logic [here](https://github.com/paypal/react-engine/blob/master/lib/client.js#L57-L60) since the `history` module is installed separately. What are your thoughts?